configs/sites/frontera/packages.yaml

```diff
@@ -0,0 +1,256 @@
+      prefix: /usr
+  bash:
+    externals:
+    - spec: bash@4.2.46
+      prefix: /usr
+  berkeley-db:
+    externals:
+    - spec: berkeley-db@5.3.21
+      prefix: /usr
+  binutils:
+    externals:
+    - spec: binutils@2.32
+      prefix: /opt/apps/gcc/9.1.0
+    - spec: binutils@2.27.44
+      prefix: /usr
+  bison:
+    externals:
+    - spec: bison@3.0.4
+      prefix: /usr
+  bzip2:
+    externals:
+    - spec: bzip2@1.0.6
+      prefix: /usr
+  cmake:
+    externals:
+    - spec: cmake@3.24.2
+      prefix: /opt/apps/cmake/3.24.2
+      modules:
+      - cmake/3.24.2
+  cpio:
+    externals:
+    - spec: cpio@2.11
+      prefix: /usr
+  curl:
+    externals:
+    - spec: curl@7.29.0+ldap
+      prefix: /usr
+  cvs:
+    externals:
+    - spec: cvs@1.11.23
+      prefix: /usr
+  diffutils:
+    externals:
+    - spec: diffutils@3.3
+      prefix: /usr
+  dos2unix:
+    externals:
+    - spec: dos2unix@6.0.3
+      prefix: /usr
+  doxygen:
+    externals:
+    - spec: doxygen@1.8.5+graphviz~mscgen
+      prefix: /usr
+  ecflow:
+    buildable: False
+    externals:
+    - spec: ecflow@5.8.4+ui+static_boost
+      prefix: /work2/06146/tg854455/frontera/spack-stack/ecflow-5.8.4
+      modules:
+      - ecflow/5.8.4
+  file:
+    externals:
+    - spec: file@5.11
+      prefix: /usr
+  findutils:
+    externals:
+    - spec: findutils@4.5.11
+      prefix: /usr
+  flex:
+    externals:
+    - spec: flex@2.5.37+lex
+      prefix: /usr
+  gawk:
+    externals:
+    - spec: gawk@4.0.2
+      prefix: /usr
+  gettext:
+    externals:
+    - spec: gettext@0.19.8.1
+      prefix: /usr
+  ghostscript:
+    externals:
+    - spec: ghostscript@9.25
+      prefix: /usr
+  git:
+    externals:
+    - spec: git@2.24.1+tcltk
+      prefix: /opt/apps/git/2.24.1
+  git-lfs:
+    externals:
+    - spec: git-lfs@2.10.0
+      modules:
+      - git-lfs/2.10.0
+      prefix: /work2/06146/tg854455/frontera/spack-stack/git-lfs/2.10.0
+  gmake:
+    externals:
+    - spec: gmake@3.82
+      prefix: /usr
+  go:
+    externals:
+    - spec: go@1.15.14
+      prefix: /usr
+  groff:
+    externals:
+    - spec: groff@1.22.2
+      prefix: /usr
+  hwloc:
+    externals:
+    - spec: hwloc@1.11.8
+      prefix: /usr
+  krb5:
+    externals:
+    - spec: krb5@1.15.1
+      prefix: /usr
+  libfuse:
+    externals:
+    - spec: libfuse@2.9.2
+      prefix: /usr
+    - spec: libfuse@3.6.1
+      prefix: /usr
+  libtool:
+    externals:
+    - spec: libtool@2.4.6
+      prefix: /opt/apps/autotools/1.2
+    - spec: libtool@2.4.2
+      prefix: /usr
+  lustre:
+    externals:
+    - spec: lustre@2.12.8_ddn7
+      prefix: /usr
+  m4:
+    externals:
+    - spec: m4@1.4.18
+      prefix: /opt/apps/autotools/1.2
+    - spec: m4@1.4.16
+      prefix: /usr
+  meson:
+    externals:
+    - spec: meson@0.55.1
+      prefix: /usr
+  ncurses:
+    externals:
+    - spec: ncurses@5.9.20130511+termlib abi=5
+      prefix: /usr
+    - spec: ncurses@6.3.20211021+termlib abi=6
```

Comment:
I would remove lines 181-182 unless you really need it. And any other package coming from miniconda

Reply:
Done